### DISCUSSION TITLE
Error Create Project Using Serin

### DISCUSSION THREAD

**Discussion by ahmadfauzanarif** [serenity-is/Serenity#6590] [Help] [answered]
Hello, I have a question.
I previously made a serenity project using visual studio code in a serin way. Previously it was running fine. But today when I created a new project, suddenly there was an error description like this. Does anyone know what this error is for? because my friends also experienced the same thing
![image](https://user-images.githubusercontent.com/71299117/203457483-3af5ef94-ea9b-4024-9d34-4e1503a54457.png)

**Comment by volkanceylan** [comment, marked as answer]
Please update serin to latest version, npm install -g serin@latest

**Reply by ahmadfauzanarif** [reply]
![image](https://user-images.githubusercontent.com/71299117/203669077-e839903a-10de-4e47-a6b9-36f529ac9fd2.png)
Thanks, it works :D
